debian/control

```diff
@@ -31,6 +31,6 @@ Depends:
  fonts-liberation,
  shared-mime-info,
  redis-server,
- screamshotter,
+ screamshotter (>= 2.0.9),
```

Comment:
Est-ce qu'il ne serait pas plus prudent de mettre également `< 3.0.0` ?

Reply:
why not